#!/usr/bin/env python
PACKAGE = "rnw_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("action_scale", double_t, 0, "action_scale", 0.5, 0, 2)
gen.add("enable_x", bool_t, 0, "enable_x",  True)
gen.add("enable_y", bool_t, 0, "enable_y",  True)

exit(gen.generate(PACKAGE, "rnw_ros", "Rl"))